modules/core/src/utils/log-to-dom.js

```diff
@@ -21,18 +21,21 @@ export function enableDOMLogging(enable = true) {
 
 let logDiv = null;
 
-export function logLineToDOM(message) {
+function logLineToDOM(opts, message) {
+  logDiv = opts.container || logDiv;
+
   if (!logDiv) {
-    const markdown = global.probe.markdown;
-    logDiv = document.createElement(markdown ? 'pre' : 'div');
+    logDiv = document.createElement('pre');
+    document.body.append(logDiv);
   }
-  // Ensure the element comes first
-  const {childNodes} = document.body;
-  document.body.insertBefore(logDiv, childNodes && childNodes[0]);
```

Comment:
Looks like there might be some change to the insert order logic here, which is fine, but just as an FYI (I don't have the details fresh), there is a bit of interaction with the canvas divs that are added by luma.gl createContext, initially the text log was hidden several screens down.

Reply:
The way we are calling `enableLogToDOM`, it should happen before any luma.gl canvas creation.

Even if it is called after canvas creation, because the screenshot calls rely on the bounding box of the canvas, I think it's reasonable to avoid arbitrarily rearranging elements in the document. I will make sure the log is visible in all our browser tests.